docs/platforms/javascript/common/best-practices/micro-frontends.mdx

```diff
@@ -76,7 +78,7 @@ which bundles may be responsible for an error. Once the destination is determine
 store it as a list of DSN-release pairs in `event.extra[MULTIPLEXED_TRANSPORT_EXTRA_KEY]`
 for the multiplexed transport to reference for routing.
 
-In practice, here is what your Sentry initialization should look like:
+**Install the below code snippet in your host:**
```

Comment:
no action required, just out of curiosity from a non-native english speaker: I wan't aware that you can say "the below code snippet". Is it equal to "the following code snippet" and should we prefer one over the other?

Reply:
Yup, they're the same :) I personally prefer "the below" because to me it sounds more specific, but they both work!